docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -499,20 +500,19 @@ const { openApp } = require('./utils/openApp');
 /* @end */
 
 describe('Home screen', () => {
-  beforeEach(async () => {
-    /* @info New line */ await openApp(); /* @end */
-  });
+beforeEach(async () => {/* @info New line */ await openApp(); /* @end */});
 
-  it('"Click me" button should be visible', async () => {
-    await expect(element(by.id('click-me-button'))).toBeVisible();
-  });
+it('"Click me" button should be visible', async () => {
+await expect(element(by.id('click-me-button'))).toBeVisible();
+});
 
-  it('shows "Hi!" after tapping "Click me"', async () => {
-    await element(by.id('click-me-button')).tap();
-    await expect(element(by.text('Hi!'))).toBeVisible();
-  });
+it('shows "Hi!" after tapping "Click me"', async () => {
+await element(by.id('click-me-button')).tap();
+await expect(element(by.text('Hi!'))).toBeVisible();
+});
 });
-```
+
+````
```

Comment:
Looks like unwanted reformat and change to four backticks at the end. Probably we need to add Prettier ignore there?

Reply:
Ah yes, missed this one and yes we need to add prettier ignore here. Will update it. Thanks for catching this and the other ones as well 👍